Add_Keyboard_Support/DriverImplementation.md

```diff
@@ -0,0 +1,217 @@
+# Handling scancodes
+
+Handling the keyboard interrupt was pretty simple, we just basically needed to add an irq to the APIC and read the scancode. But at the end what we got is just a number, and we want to eventually translated it into a character. 
```

Comment:
Perhaps this could rephrased a little. The ps2 controller is a system device, so it's irq line was enabled in the ioapic, then an entry was added in the idt. Now inside of the interrupt handler we just added we can read the scancode from the keyboard.